ztree-diff.el

```diff
@@ -95,6 +95,8 @@ By default paths starting with dot (like .git) are ignored")
 (defvar-local ztree-diff-wait-message nil
   "Message showing while constructing the diff tree.")
 
+(defvar ztree-diff--ediff-previous-window-configuration nil
```

Comment:
Done.

You *should* use double dashes for private variables/functions--it's Emacs convention, and helps keep the interface small.